music_assistant/helpers/scrobbler.py

```diff
@@ -0,0 +1,82 @@
+"""Helper class to aid scrobblers."""
+
+import asyncio
+import logging
+
+from music_assistant_models.event import MassEvent
+from music_assistant_models.playback_progress_report import MediaItemPlaybackProgressReport
```

Comment:
If these are only used for type checking, they should be moved into a "if TYPE_CHECKING" block
You can import the TYPE_CHECKING constant from the typing library

Reply:
Done. Ruff disliked it until I added `from __future__ import annotations` too. Since I see that elsewhere in the code base I assume that's fine.